### PR TITLE
Softmax function

### PR DESCRIPTION
**Pull request by emregoynugur**
The implementation is not very efficient and it doesn't fully support GPU, because we lack some functionality. However, it seems to work so far :)

I or someone else can add `autograd` functionality in a different PR. Please do review!

### CODE REVIEW

File: UnityProject/Assets/OpenMined.Tests/Editor/FloatTensor/FloatTensorTest.cs [hunk 1]
```diff
@@ -1807,6 +1812,97 @@ public void SizeTensor()
             }
         }*/
 
+        [Test]
+        public void Softmax1D()
```
Comment (bendecoste) [inline UnityProject/Assets/OpenMined.Tests/Editor/FloatTensor/FloatTensorTest.cs]: there are commented out tests in the functional dir that you could uncomment/move these to

Comment (emregoynugur) [reply UnityProject/Assets/OpenMined.Tests/Editor/FloatTensor/FloatTensorTest.cs]: do you mean Softmax.cs? I haven't added it but I can remove it if it is okay.

Comment (emregoynugur) [reply UnityProject/Assets/OpenMined.Tests/Editor/FloatTensor/FloatTensorTest.cs]: gone

**Comment by emregoynugur** [comment]
For some reason, it is `Sigmoid` functions, which are failing. It seems that `children_indices` is null. I shouldn't have changed anything that can break this functionality.

Anyone have an idea?

**Pull request merged**